iceoryx_hoofs/test/moduletests/test_utility_convert.cpp

```diff
@@ -42,10 +71,19 @@ class convert_test : public Test
     template <typename T>
     std::string fp_to_string(T value)
     {
-        static_assert(std::is_floating_point<T>::value, "fp_to_string requires floating point type");
+        static_assert(std::is_floating_point<T>::value, "requires floating point type");
 
         std::ostringstream oss;
-        oss << std::scientific << std::setprecision(std::numeric_limits<T>::max_digits10) << value;
+        oss << std::scientific << std::setprecision(std::numeric_limits<T>::digits10) << value;
+        return oss.str();
+    }
+    template <typename T>
+    std::string fp_to_string(T value, uint16_t digits)
```

Comment:
Just as food for thought. The two `fp_to_string` methods can be combined like this
```suggestion
    std::string fp_to_string(T value, uint16_t digits = std::numeric_limits<T>::digits10)
```